### PR TITLE
Added link to the forum to the duck.co header

### PR DESCRIPTION
**Pull request by alohaas**
Added a link to the new forum to duck.co header navigation.

cc: @russellholt @MariagraziaAlastra

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
@alohaas Thanks! 😄 
Could you please apply the changes [here](https://github.com/duckduckgo/community-platform/blob/master/templates/header_nav.tx) as well? Otherwise the link will only be visible from the blog!

**Comment by alohaas** [comment]
@MariagraziaAlastra sure! thanks for the heads up.

**Comment by alohaas** [comment]
@MariagraziaAlastra and @jbarrett both nav templates have been updated with the new forum link.

**Comment by MariagraziaAlastra** [comment]
Thanks @alohaas, LGTM!
Will merge after the release on Monday, so it can go live on Tuesday :smile:

File: templates/header_nav.tx [hunk 1]
```diff
@@ -2,9 +2,10 @@
 	<ul>
 		<!-- <li class="nav-home"><a href="<: $u('Root','index') :>">Home</a></li> -->
 		<li class="nav-develop"><a href="/ia/dev">Instant Answers</a></li>
-		<li class="nav-docs"><a href="http://docs.duckduckhack.com">Docs</a></li>
+		<li class="nav-docs"><a href="https://docs.duckduckhack.com">Docs</a></li>
```
Comment (jbarrett) [inline templates/header_nav.tx]: I don't think we've got a cert for docs.ddh yet.

**Comment by alohaas** [comment]
@jbarrett @MariagraziaAlastra I removed the `S's`. Should be good to go.

**Comment by MariagraziaAlastra** [comment]
:+1: Thanks @alohaas!

**Pull request merged**